compiler/modules/front/astString.bal

```diff
@@ -0,0 +1,217 @@
+import wso2/nballerina.err;
+
+
+// zero-width joiner
+const ZWJ = ();
```

Comment:
ZWJ has a well-established meaning as U+200D, which is not what you mean here, so choose another name.